doc/spec.md

```diff
@@ -1193,7 +1193,7 @@ The [type](#type) of a built-in function is `"builtin_function_or_method"`.
 A built-in function value used in a Boolean context is always considered true.
 
 Many built-in functions are predeclared in the environment
-(see [Name Resolution](#name-resolution)).
+(see [Built-in constants and functions](#built-in-constants-and-functions)).
```

Comment:
I think the previous link was in support of the concept of "predeclared". How about:

```
Many [built-in functions](#built-in-constants-and-functions) are
[predeclared](#name-resolution) in the environment.
```
?

Or remove the first link and keep the "(see...)" remark as you have it.

Reply:
Thanks for the comment. 

I think it would be better to add links as much as we can because it can improve the users' reading experience. 

Yous proposed change still has link `#name-resolution`, which doesn't exist at all. I guess you wanted to say `#name-binding-and-variables`?

Just updated. PTAL, thx